modules/BMP280.js

```diff
@@ -9,7 +9,7 @@ function BMP280() {
     var i2c = require("i2c");
     bmp280API.i2cDevice = i2c.open({ bus: 0, speed: 100 });
 
-    var bmp280 = {
+    bmp280API.bmp280Addrs = {
```

Comment:
see my comments above, if you expose the setDisplayState and setFunction, then you don't need to expose the I2C calls.